src/SOS/SOS.UnitTests/Scripts/GCTests.script

```diff
@@ -23,6 +23,15 @@ IFDEF:CDB
 VERIFY:Break instruction exception - code 80000003
 ENDIF:CDB
 
+SOSCOMMAND:DumpStackObjects
+VERIFY:<HEXVAL>\s+<HEXVAL>\s+System.IO.StringWriter\s+
+
+SOSCOMMAND:DumpObj <POUT>\w+\s+(<HEXVAL>)\s+(System.IO.StringWriter!\$0_)*System.IO.StringWriter\s+<POUT>
+IFDEF:MAJOR_RUNTIME_VERSION_2
```

Comment:
Is this only for 2.x tests?

Reply:
I forgot to add the VERIFY for 3.0; it is a different format. 1.1/2.1 are both the same.  I'll add some thing for both.